docs/recipes/Deploying-Modules.md

```diff
@@ -0,0 +1,55 @@
+[👈 Return to Overview](./Recipes.md)
+
+# Deploying Modules
+
+## 📖 Table of Contents
+
+* [Deploying to Now with GitHub Actions](#deploying-to-now-with-github-actions)
+
+# Deploying to Now with GitHub Actions
+
+[Now.sh](https://zeit.co/home) allows you to deploy statics such as your modules. We will accomplish this be using [GitHub Actions](https://github.com/features/actions).
+
+## Creating your Deploy Action
+
+This deploy action uses Zeit's `now-cli` which you will need a [token](https://zeit.co/account/tokens) from Now. The below action runs whenever a release is created or a push to master. This will create 
```

Comment:
For this you will need Zeit's [now-cli](link-to-it-please) and a [token from Now](https://zeit.co/account/tokens).